packages/cli/package.json

```diff
@@ -97,6 +98,7 @@
     "fs-jetpack": "5.1.0",
     "get-port": "5.1.1",
     "global-dirs": "3.0.1",
+    "http": "0.0.1-security",
```

Comment:
```suggestion
```
`http` comes from Node.js, it’s not a package.

Reply:
Totally my bad. Thanks for spotting!